alpha/lib/model/entry.php

```diff
@@ -2965,8 +2965,19 @@ public function getCacheInvalidationKeys()
 	/**
 	 * @return entry
 	 */
-	public function copyTemplate($coptPartnerId = false)
+	public function copyTemplate($coptPartnerId = false, $entry = null)
 	{
+		if ($entry)
+		{
+			$templateType = $this->getType();
+			$type = $entry->getType();
+			$templateMediaType = $this->getMediaType() ? $this->getMediaType() : "null";
+			$mediaType = $entry->getMediaType() ? $entry->getMediaType() : "null";
+	
+			if ($templateType != $type || $templateMediaType != $mediaType || $templateExternalSourceType != $externalSourceType)
```

Comment:
@shpiNi, both $templateExternalSourceType and $externalSourceType are not defined.